Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:16.3 as nodebuild
+COPY ./frontend /frontend
+WORKDIR /frontend
+RUN rm -rf node_modules
+RUN npm install
+RUN npm run build
+FROM quay.io/bitnami/python
```

Comment:
Can you add an extra line in your multi-stage build, just so it's easier to read?
Also, we should probably be using centos:8 instead of this one, since it's what we're using for the rest of our projects

Reply:
Is there an image of centos python in quay.io that I can use? I notice that we're using `FROM registry.centos.org/centos/centos:8` in quay project, do i need to use it and install python3 additionally?